rave-compiler/src/main/java/com/uber/rave/compiler/RaveWriter.java

```diff
@@ -97,6 +99,9 @@ public void generateJava(@NonNull RaveIR raveIR) throws IOException {
         String className = raveIR.getSimpleName() + GENERATED_CLASS_POSTFIX;
         TypeSpec.Builder builder = TypeSpec.classBuilder(className);
         builder.superclass(BaseValidator.class)
+                .addAnnotation(AnnotationSpec.builder(Generated.class)
+                        .addMember("value", "RaveProcessor")
```

Comment:
Make this the fully qualified name of the code generator as recommended in the [docs](https://docs.oracle.com/javase/7/docs/api/javax/annotation/Generated.html).

A comment indicating this validator is generated for all models annotated with `@Validated` in this module/library would also be helpful.